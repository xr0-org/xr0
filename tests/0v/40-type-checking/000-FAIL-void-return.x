void
foo()
{
	return 1;
}
