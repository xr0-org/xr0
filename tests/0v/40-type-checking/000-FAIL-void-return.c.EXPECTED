40-type-checking/000-FAIL-void-return.x:4:10: cannot return int as void (foo)
